invenio_theme/assets/semantic-ui/less/invenio_theme/theme/collections/grid.overrides

```diff
@@ -1,3 +1,36 @@
 /*******************************
          Site Overrides
 *******************************/
+
+.row.no-padding,
+.row .column.no-padding {
+    padding: 0px !important;
+}
+
+.row.no-padding-lr,
+.row .column.no-padding-lr {
+    padding-left: 0px !important;
+    padding-right: 0px !important;
+}
+
+.row.no-padding-tb,
+.row .column.no-padding-tb {
+    padding-top: 0px !important;
+    padding-bottom: 0px !important;
+}
+
+/** Deposit */
+.row.deposit-result-header {
```

Comment:
Will merge this and investigate if this can be imported in invenio-app-rdm instead as an rdm theme.